.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
This only ignores the top-level `node_modules`, doesn't it? We have more `node_modules` per package.

Reply:
Yes that's right. The other node_modules are very small (< 500k) compared to the big one in root. But maybe it's better to exclude them also.

Reply:
```suggestion
**/node_modules
```